<launch>

  <arg name="input_image" default="/image_raw" />
  <arg name="input_projected_roi" default="/feat_proj_bbox/output_projected_roi" />
  <arg name="input_tracker_roi" default="/tlr_kcf_tracker/output_rois" />
  <arg name="input_signals" default="/roi_signal_proj" />
  <arg name="input_light_state" default="/tlr_classifier/output_light_states" />

  <node name="tlr_visualizer"
        pkg="trafficlight_recognizer" type="tlr_visualizer"
        output="screen">
    <remap from="~input_image" to="$(arg input_image)" />
    <remap from="~input_projected_roi" to="$(arg input_projected_roi)" />
    <remap from="~input_tracker_roi" to="$(arg input_tracker_roi)" />
    <remap from="~input_signals" to="$(arg input_signals)" />
    <remap from="~input_light_state" to="$(arg input_light_state)" />
    <rosparam subst_value="true">
      approximate_sync: true
    </rosparam>

  </node>

</launch>
